src/backer/cardaning.py

```diff
@@ -24,6 +26,9 @@
 FUNDING_AMOUNT = os.environ.get('FUNDING_AMOUNT', 30000000)
 TRANSACTION_AMOUNT = os.environ.get('TRANSACTION_AMOUNT', 1000000)
 MIN_BLOCK_CONFIRMATIONS = os.environ.get('MIN_BLOCK_CONFIRMATIONS', 3)
+SHELLY_UNIX = os.environ.get('SHELLY_UNIX', 1666656000) #"2022-10-25T00:00:00Z"
+TRANSACTION_DEEP = os.environ.get('TRANSACTION_DEEP', 16)
+TRANSACTION_CONFIRMATION_TIMEOUT = os.environ.get('TRANSACTION_CONFIRMATION_TIMEOUT', 30)
```

Comment:
Hmm, 30 seconds to get 16 blocks deep won't work. You need like 5 minutes to be sure. What value have you been testing with?

Reply:
:) It's my bad, It must be 30 mins

Reply:
But in the test, We can set 30 seconds test test that function :)